src/wagtail_ai/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from .prompts import Prompt as PromptDataClass
+
+
+class Prompt(models.Model):
```

Comment:
**Suggestion:** We could add RevisionMixin and LockableMixin. I think it'd be beneficial for editors to have that available.